server/src/main/scala/za/co/absa/atum/server/api/database/Runs.scala

```diff
@@ -33,6 +33,7 @@ class Runs (implicit dBEngine: SlickPgEngine) extends DBSchema{
   import Runs._
 
   val writeCheckpoint = new WriteCheckpoint
+  val createPartitioningIfNotExists = new CreatePartitioningIfNotExists
 }
 
 object Runs {
```

Comment:
Method `nestedScalaSeqToPgArray` is not more used.

Reply:
Yeah neither is the one below, but I think it will be, once we'll use TEXT[] or TEXT[][] fields.

Reply:
I don't think we will ever use `TEXT[][]` becuse of the subarrays` limitation (all the same size).
This one can be safely removed.

Reply:
done